app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Look at the subjects you could teach
+description: |-
+  Look at the subjects you could teach and choose what teacher training course you would like to do.
+
+keywords:
+  - teacher training
+  - teaching maths
+  - teaching history
+  - teaching science
+  - teaching art
+  - art and design
+
+
+---
+
+Explore the A-Z of all the subjects you could choose for your teacher training course.
```

Comment:
@caitlin-reidy it might be an idea to ask Jen to look at the design of the page to see if she has any suggestions for how it could look? Would we want to have A then list all the a subjects e.t.c and use anchor links to make subjects easier for people to find so they don't have to scroll through all of them? Or do we want people to see the breadth of subjects available to train in? - question for Gemma?